geopandas/geodataframe.py

```diff
@@ -741,9 +745,22 @@ def to_json(self, na="null", show_bbox=False, drop_id=False, **kwargs):
         GeoDataFrame.to_file : write GeoDataFrame to file
 
         """
-        return json.dumps(
-            self._to_geo(na=na, show_bbox=show_bbox, drop_id=drop_id), **kwargs
-        )
+        geo = self._to_geo(na=na, show_bbox=show_bbox, drop_id=drop_id)
+
+        if self.crs is not None:
+            auth_crsdef = self.crs.to_authority()
+            if auth_crsdef is None:
+                warnings.warn(
+                    "Geodataframe CRS is not representable in URN OGC "
+                    "format. Resulting JSON will bear no CRS information.",
```

Comment:
```suggestion
                    "format. Resulting JSON will contain no CRS information.",
```
Splitting hairs, I think avoiding 'bear' is slightly clearer